build.gradle

```diff
@@ -71,6 +71,8 @@ configurations {
     shaded
     // Place shaded dependencies into `compileOnly` so that they do not leak into our publications' dependencies.
     compileOnly.extendsFrom shaded
+    testCompileOnly.extendsFrom shaded
```

Comment:
This is a bit cursed. Now the tests compile against the non-shaded version, but if they use the plugin they will be using the shaded version. I suppose that is fine.